Data/Csv.hs

```diff
@@ -178,9 +190,13 @@ import Data.Csv.Types
 -- Derived:
 --
 -- > {-# LANGUAGE DeriveGeneric #-}
+-- > {-# LANGUAGE OverloadedStrings #-}
```

Comment:
`Show` for sake of `decode` in `ghci`.
`name'` and `age'` because `ghci` complains `This binding for ‘name’ shadows the existing binding`.
`OverloadedStrings` will be removed.